src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.courses.domain;
+
+import nextstep.users.domain.NsUser;
+
+import java.util.List;
+
+import static nextstep.courses.domain.SessionStatus.RECRUITING;
+
+public abstract class Session {
+
+    protected final String sessionId;
+    protected final SessionDate date;
+    protected final SessionImage image;
+    protected final SessionStatus status;
+    protected List<NsUser> students;
+
+    public Session(String sessionId, SessionDate date, SessionImage image, SessionStatus status, List<NsUser> numOfStudents) {
+        this.sessionId = sessionId;
+        this.date = date;
+        this.image = image;
+        this.status = status;
+        this.students = numOfStudents;
+    }
+
+    public void enroll(NsUser user) {
+        if (!RECRUITING.equals(status)) {
```

Comment:
메세지를 보내어 처리할 수 있을것 같아요 :)

```suggestion
        if (!status.canEnroll()) {
```

Reply:
맞네요! enum 내부에 메소드 구현했습니다!